test/test.cpp

```diff
@@ -88,7 +88,7 @@ TEST(Retriever, http)
   try
   {
     Retriever r;
-    MemoryResource res = r.get("http://pr.willowgarage.com/downloads/svnmerge.py");
+    MemoryResource res = r.get("http://repositories.ros.org/repos.key");
```

Comment:
That is not a documented URL and should not be considered stable. I'd recommend `http://packages.ros.org/ros.key` instead.